src/backend/optimizer/plan/createplan.c

```diff
@@ -9063,4 +9064,148 @@ push_locus_down_after_elide_motion(Plan* plan)
 				plan = plan->lefttree;
 		}
 	}
-}
```

Comment:
add a new line.

Reply:
but add a newline won't get rid of this diff. this diff is due to *the lack of* a newline at the end of file.